components/flow-orchestration-framework/org.wso2.carbon.identity.flow.execution.engine/src/main/java/org/wso2/carbon/identity/flow/execution/engine/FlowExecutionService.java

```diff
@@ -140,6 +140,13 @@ public FlowExecutionStep executeFlow(String tenantDomain, String applicationId,
 
     private boolean enterFlowInIdentityContext(String flowType) {
 
+        if (StringUtils.isBlank(flowType) ||
+                Arrays.stream(org.wso2.carbon.identity.flow.mgt.Constants.FlowTypes.values())
```

Comment:
Without using Arrays.stream, can't you use EnumUtils? Which is tend to fault tolerant.

EnumUtils.isValidEnumIgnoreCase(Constants.FlowTypes.class, flowType)
EnumUtils.getEnumIgnoreCase(Constants.FlowTypes.class, flowType)